java/api/src/main/java/org/ray/api/function/PyActorClass.java

```diff
@@ -0,0 +1,14 @@
+package org.ray.api.function;
+
+/**
+ * A class for creating Python actor.
+ */
+public class PyActorClass {
+  public String moduleName;
+  public String className;
```

Comment:
`public final` to make it immutable.

Reply:
And document the public fields?

Reply:
I think the field name is clear enough. Should I add document?

Reply:
> `public final` to make it immutable.

I will add public final